src/linux/system.rs

```diff
@@ -777,6 +777,7 @@ fn get_exe_name(p: &Process) -> String {
         .unwrap_or_else(String::new)
 }
 
+#[derive(PartialEq)]
```

Comment:
I guess that this change comes from the fix? ;)

Reply:
Removed.